### DISCUSSION TITLE
How can I install Positron on Windows ARM64?

### DISCUSSION THREAD

**Discussion by denis-or** [posit-dev/positron#5249] [Q&A]
Hi everyone,

I recently acquired an Asus Vivobook S15 with Windows 11 Home. This laptop has an ARM64 processor (Snapdragon X Elite).

Many resources I’ve tried to use are being installed and seem to work (through emulation, I assume).

However, when attempting to install Positron, I received the following message and couldn’t complete the installation:
![Captura de tela 2024-11-02 141933](https://github.com/user-attachments/assets/cac4808e-edce-4949-a8e5-92635002133a)

In English, it means that the installation can only be completed on supported processor versions.

I couldn’t find anything about Windows ARM64 in the issues or here, so I’d appreciate any help.

**Comment by lionel-** [comment]
Unfortunately we currently don't build Positron for arm64 Windows. I've created https://github.com/posit-dev/positron/issues/5254 to track progress on this.
